src/app/zap-templates/templates/app/cluster-objects.zapt

```diff
@@ -60,11 +60,23 @@ using Fields = Clusters::detail::Structs::{{asUpperCamelCase name}}::Fields;
 
 struct Type : public Clusters::detail::Structs::{{asUpperCamelCase name}}::Type
```

Comment:
add a zap comment on usage of these classes?

My reading instinct here is ask `why do we have an empty strct` and following that `why do we even care about constructors of empty structs`.

Something like "make the type be type-safe so do not use `using`, constructors is to make aliasing easier" or something similar.

Reply:
Will add.

Reply:
Done.